client/src/components/LogStatusButton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+// Components
+import LogStatus from './LogStatus';
+
+// Styles
+import styles from './../sass/components/LogStatusButton.module.scss';
```

Comment:
This is way down the list of priorities but it would be nice to have path aliases set up so we can clean up the imports and get rid of './../../../' etc. As I said, probably not important for this project but definitely something to be aware of for the future

Reply:
@maxckelly can we merge this branch?

Reply:
@mark-tomlinson-dev - How do we do that? Unless we remove some folders and change the structure of the files.